x-pack/plugins/stack_connectors/server/connector_types/bedrock/bedrock.test.ts

```diff
@@ -333,7 +333,27 @@ describe('BedrockConnector', () => {
 
         // @ts-expect-error expects an axios error as the parameter
         expect(connector.getResponseErrorMessage(err)).toEqual(
-          `Unauthorized API Error - The api key was invalid.`
+          `Unauthorized API Error: The api key was invalid.`
+        );
+      });
+
+      it('returns endpoint error', () => {
+        const err = {
+          response: {
+            headers: {},
+            status: 400,
+            statusText: 'Bad Request',
+            data: {
+              message: 'The requested operation is not recognized by the service.',
+            },
+          },
+        } as AxiosError<{ message?: string }>;
+
+        // @ts-expect-error expects an axios error as the parameter
+        expect(connector.getResponseErrorMessage(err)).toEqual(
+          `API Error: The requested operation is not recognized by the service.
```

Comment:
Seems slightly weird to have an multi-line error message - or at least non-standard.  I would guess that in some environments (writing to the Kibana log) these will get removed/replaced - for instance, in the server log connector, we replace "`\n`" with "`; `". 

Does it **need** to be multi-line?

Reply:
It doesn't need to, it just looks nicer in the assistant. If you look at the screenshots in the description, you can see it puts it all in one line anyways on the server, so it shouldn't matter. I have a [comment about it in the code](https://github.com/elastic/kibana/pull/176090/files#diff-f0215878b5d4d091a224b60506563a1872aa30f9df48d8a8a000b40b331ca758R96). Is this ok?

Reply:
I suspect it will be fine, but as it's not what we typically do (AFAIK), there could be some unintended side effects - I would guess in UX's. Have you checked what this looks like in the Kibana logs as displayed in cloud log viewers?  I don't >think< it will split this into multiple log documents ...

Reply:
I don't think so either, but will deploy this branch to double check

Reply:
One document, keeps the space:

<img width="866" alt="Screenshot 2024-02-02 at 10 11 24 AM" src="https://github.com/elastic/kibana/assets/6935300/d5020cb1-21a7-4a45-ac27-2b4f90a63207">